concrete-core-test/src/core.rs

```diff
@@ -54,5 +54,6 @@ test! {
     (GlweCiphertextEncryptionFixture, (PlaintextVector, GlweSecretKey, GlweCiphertext)),
     (LweCiphertextVectorDiscardingAffineTransformationFixture, (LweCiphertextVector, CleartextVector, Plaintext, LweCiphertext)),
     (LweCiphertextDiscardingKeyswitchFixture, (LweKeyswitchKey, LweCiphertext, LweCiphertext)),
-    (PlaintextCreationFixture, (Plaintext))
+    (PlaintextCreationFixture, (Plaintext)),
+    (LweCiphertextDiscardingAdditionFixture, (LweCiphertext, LweCiphertext))
```

Comment:
Here I would just try to keep the LweCiphertext tests together

Reply:
Better? I guess at some point we should then also sort the GLWE and LWE tests.